src/routes/webhooks/paddle.ts

```diff
@@ -748,23 +748,37 @@ export const updateUserTransaction = async ({
   });
 
   if (!transaction) {
-    const newTransaction = await con
+    const insertResult = await con
       .getRepository(UserTransaction)
-      .save(payload);
-
-    return newTransaction;
+      .createQueryBuilder()
+      .insert()
+      .values(payload)
+      .onConflict(
+        `((flags->>'providerId')) DO UPDATE SET status = EXCLUDED.status,
+            value = EXCLUDED.value,
+            "valueIncFees" = EXCLUDED."valueIncFees",
+            "updatedAt" = NOW()`,
+      )
+      .returning(['id'])
+      .execute();
+
+    payload.id = insertResult.raw[0].id;
+
+    return payload;
   } else {
     await con.getRepository(UserTransaction).update(
       { id: transaction.id },
       {
         value: itemData.price.customData.cores,
+        valueIncFees: itemData.price.customData.cores,
```

Comment:
Also fixed issue where I did not update `valueIncFees` if user selected different cores pack